### PR TITLE
dependency-check 1.0.0

### PR DESCRIPTION
**Pull request by connor-alphero**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/dependency-check/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,154 @@
+title: Dependency Check
```
Comment (ofalvai) [inline steps/dependency-check/1.0.0/step.yml:1]: This title seems a bit too generic, how about `OWASP Dependency Check`?

File: steps/dependency-check/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+source:
+  git: https://github.com/alpheroltd/bitrise-step-dependency-check.git
+  commit: 24c466c0e62c363b2cf6bd2378b5b4656adb51f3
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/alpheroltd/bitrise-step-dependency-check
+deps:
+  brew:
```
Comment (ofalvai) [inline steps/dependency-check/1.0.0/step.yml:18]: Is it supposed to run on Linux VMs too? If so, you can define an `apt` dependency here (assuming the tool is available in APT)

https://devcenter.bitrise.io/en/builds/installing-tools-during-a-build.html#installing-tools-by-declaring-deps-in-the-bitrise-yml-file

File: steps/dependency-check/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    value_options:
+    - "yes"
+    - "no"
+- data_path: /tmp/dependancy-check-cache
+  opts:
+    category: Config
+    description: |
+      Description of this input.
+
+      Can be Markdown formatted text.
```
Comment (ofalvai) [inline steps/dependency-check/1.0.0/step.yml:69]: Some leftovers from the step template :)

**Review by ofalvai — CHANGES_REQUESTED** [review]
Great work @connor-alphero, thank you for developing this step. I left a few minor comments on the step.yml metadata.

**Comment by connor-alphero** [comment]
@ofalvai Thank you for the feedback. 

I will incorporate what you pointed out and will open a fresh PR soon.

**Pull request closed without merging**